sympy/solvers/tests/test_ode.py

```diff
@@ -2333,13 +2333,13 @@ def test_nth_linear_constant_coeff_variation_of_parameters_simplify_False():
     # eq.subs(*sol_simp.args) doesn't simplify to zero without help
     (t, zero) = checkodesol(eq, sol_simp, order=5, solve_for_func=False)
     # if this fails because zero.is_zero, replace this block with
-    # assert checkodesol(eq, sol_simp, order=5, solve_for_func=False)[0]
-    assert not zero.is_zero and zero.rewrite(exp).simplify() == 0
+    assert checkodesol(eq, sol_simp, order=5, solve_for_func=False)[0]
+    # assert not zero.is_zero and zero.rewrite(exp).simplify() == 0
     # \-----------
     (t, zero) = checkodesol(eq, sol_nsimp, order=5, solve_for_func=False)
     # if this fails because zero.is_zero, replace this block with
-    # assert checkodesol(eq, sol_simp, order=5, solve_for_func=False)[0]
-    assert zero == 0
+    assert checkodesol(eq, sol_simp, order=5, solve_for_func=False)[0]
+    # assert zero == 0
```

Comment:
If `checkodesol` now works for this case we can clean this up and remove all of those comments.

Reply:
I'd prefer this though:
```
assert checkodesol(...) == (True, 0)
```